theseus/core/cost_function.py

```diff
@@ -37,6 +66,13 @@ def __init__(
         super().__init__(name=name)
         self._weight = cost_weight
 
+        # For now this is only supported inside vectorized cost functions
+        # When set to true, `weighted_jacobians_error()` first queries the
+        # weight to check indices of zero elements, then uses
+        # `masked_jacobians()` to evaluate jacobians only over non-zero
+        # elements.
+        self.__supports_masking__ = False
```

Comment:
If I understand correctly, we can use this feature to mask some of the optimization problems that have already converged?

Reply:
Yeah, I think we can find a way to use this to address #395.